scss/_variables.scss

```diff
@@ -318,9 +318,22 @@ $container-max-widths: (
 // Set the number of columns and specify the width of the gutters.
 
 $grid-columns:                12 !default;
-$grid-gutter-width:           30px !default;
+$grid-gutter-width:           2rem !default;
```

Comment:
Thoughts on moving to `1.5rem` here? Feels a little too spaced out at certain sizes right now.